GLEAM_MEASUREMENT_DEVICE/GLEAM_OLED.ino

```diff
@@ -103,6 +104,11 @@ void updateDisplay() {
     }
   }
 
+    //Macy: OLED Displays sleep if the button is not pressed for 60 seconds
+  else if ((millis()-pressTime)> 60000) {

```

Comment:
I believe this else is referring to the if on line 46, which I think will almost always be true